Source/SLKTextViewController.m

```diff
@@ -1153,7 +1153,7 @@ - (void)slk_performRightAction
 {
     NSArray *actions = [self.rightButton actionsForTarget:self forControlEvent:UIControlEventTouchUpInside];
     
-    if (actions.count > 0 && [self canPressRightButton]) {
```

Comment:
This was not allowing us to send empty messages, even though we need to support that for our precanned messages. The button being enabled is already being controlled by ```[self canPressRightButton```, so this is redundant